.github/workflows/install-package.yml

```diff
@@ -0,0 +1,27 @@
+name: Test Installation
+
+on: [workflow_dispatch, push, pull_request]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.8']
+
+    name: Check installation with python ${{ matrix.python-version }} on ${{ matrix.os }}
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Update pip
+      run: pip install --upgrade pip
+
+    - name: Install package
+      run: python -m pip install -e .[test]
```

Comment:
Do you need to install the [test] extra here?  If simply making sure sectionproperties itself can install, then it probably isn't needed.

Reply:
Noted